go.mod

```diff
@@ -27,13 +27,13 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/xeipuuv/gojsonschema v1.2.0 // indirect
 	go.uber.org/zap v1.11.0 // indirect
-	k8s.io/api v0.20.0
-	k8s.io/apimachinery v0.20.0
+	k8s.io/api v0.21.0-rc.0
+	k8s.io/apimachinery v0.21.0-rc.0
```

Comment:
It seems that the openshift/api bump _tries_ to pull in the rebase to k8s 1.21. This doesn't actually happen in this PR because of the replaces below.

Reply:
Hmm, should we drop the replaces?  Who is responsible for fixing that?